cmd/kubeadm/app/discovery/discovery.go

```diff
@@ -37,7 +37,7 @@ func For(cfg *kubeadmapi.NodeConfiguration) (*clientcmdapi.Config, error) {
 	// we also need an ability for the user to configure the client to validate received CA cert against a checksum
 	clusterinfo, err := GetValidatedClusterInfoObject(cfg)
 	if err != nil {
-		return nil, fmt.Errorf("couldn't validate the identity of the API Server: %v", err)
+		return nil, fmt.Errorf("Couldn't validate the identity of the API Server: %v", err)
```

Comment:
having error messages is actually intended, because they are gonna be wrapped in any case...
Please change these two back